php/Terminus/UserMembership.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Terminus;
+use \ReflectionClass;
+use \Terminus\Request;
+use \Terminus\Collections\Bindings;
+use Terminus\Collections\Workflows;
+
+class UserMembership {
+  private $id;
+  private $attributes;
+  private $site;
+  private $user;
+  private $workflows;
+
+  /**
+   * Object constructor
+   *
+   * @param [stdClass] $attributes
+   * @param [array]    $options
+   * @return [UserMembership] $this
+   */
+  public function __construct($attributes, $options = array()) {
+    if(!is_array($options)) {
+      $options = get_object_vars($options);
+    }
+    foreach($options as $var_name => $value) {
+      $this->$var_name = $value;
+    }
+    $this->attributes = $attributes;
+    $this->workflows = new Workflows(array('owner' => $this->site, 'owner_type' => 'site'));
```

Comment:
I would rather that we invoke workflows from the site property, than attach a collection of site workflows to this object.